plugins/woocommerce-blocks/assets/js/blocks/product-collection/edit/inspector-controls/index.tsx

```diff
@@ -154,22 +165,20 @@ const ProductCollectionInspectorControls = (
 					<FilterableControl { ...queryControlProps } />
 				) }
 				<LayoutOptionsControl { ...displayControlProps } />
+				<WidthOptionsControl { ...dimensionsControlProps } />
 				<ColumnsControl { ...displayControlProps } />
+				{ showProductsPerPageControl && (
+					<ProductsPerPageControl { ...queryControlProps } />
+				) }
 				{ showOrderControl && (
 					<OrderByControl { ...queryControlProps } />
 				) }
```

Comment:
Sorry for being picky here but I'd suggest changing the controls order so the range controls are next to each other. I think it's easier to digest the information if the same type of controls are next to each other

```suggestion
				{ showProductsPerPageControl && (
					<ProductsPerPageControl { ...queryControlProps } />
				) }
				<ColumnsControl { ...displayControlProps } />
				{ showOrderControl && (
					<OrderByControl { ...queryControlProps } />
				) }
```

Current | My suggestion
<img width="283" alt="image" src="https://github.com/user-attachments/assets/a7c73132-7f6d-4f68-a5bd-6f608feb4136"> | <img width="271" alt="image" src="https://github.com/user-attachments/assets/fb7c2d67-7d0d-480e-a129-0ccaad80afeb">

Reply:
Thanks Karol! I appreciate your attention to detail. I have made the changes in 804041c